<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainWindow</class>
 <widget class="QMainWindow" name="QMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="convert_bt">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>540</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QLabel" name="percentage_l">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>476</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="percentage_sp">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>473</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_bt">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>539</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_bt">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>539</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>제거</string>
    </property>
   </widget>
   <widget class="QComboBox" name="direction_cb">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>470</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>좌 우</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>상 하</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>상하좌우</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="save_bt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>539</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save location</string>
    </property>
   </widget>
   <widget class="QListWidget" name="pic_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>201</width>
      <height>471</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="save_loc_lb">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Convert전 저장위치 선택 필수!</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_show_l">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="progress_show_l">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>430</y>
      <width>420</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Not started</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_show_l">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>476</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
